src/Commands/App/Builder.php

```diff
@@ -200,6 +201,10 @@ protected function prepare(): Builder
             File::put($file, '<?php return ' . var_export($config, true) . ';' . PHP_EOL);
         });
 
+        $this->task('Removing dev dependencies', function () {
+            ($process = new Process("composer install --no-dev", $this->app->basePath()))->run();
```

Comment:
why are you saving the `$process` variable here?

Reply:
@svenluijten It's ok. What matters is the idea behind @mpociot pull request.

So, now I will probably move the composer install method logic to the class `LaravelZero\Framework\Providers\Composer` anyway. Because otherwise I can't mock this for testing.

Thank you both.